.github/workflows/tf-nightly.yml

```diff
@@ -23,36 +11,48 @@ jobs:
     steps:
       - name: Checkout repository
         uses: actions/checkout@v4
-        with:
-          fetch-depth: 0
 
       - name: Configure AWS Credentials
         uses: aws-actions/configure-aws-credentials@v4
         with:
-          role-to-assume: ${{ secrets.aws_role }}
+          role-to-assume: arn:aws:iam::249446771485:role/tf-role-arc-generic
           aws-region: us-west-2
 
       - name: Install Lacework CLI
         shell: bash
         run: |
           curl https://raw.githubusercontent.com/lacework/go-sdk/main/cli/install.sh | bash
           lacework version
+          jq -n \
+          --arg ac "$LW_ACCOUNT" \
+          --arg ke "$LW_API_KEY" \
+          --arg se "$LW_API_SECRET" \
+          --arg su "$LW_SUBACCOUNT" \
+          '{keyId: $ke, secret: $se, subAccount: $su, account: $ac}' > creds.json
+          lacework configure -j creds.json --noninteractive 
```

Comment:
I may be missing context here, but why bother configuring this? Can't we use use the ENV vars and not needed creds.json?

Reply:
I couldn't get a newly install lacework cli to work without running configure. I tried with the env vars.

Reply:
To be specific I couldn't get the lacwork terraform provider to initialize properly without a configured cli